src/tn_sdk/exceptions/exceptions.py

```diff
@@ -0,0 +1,21 @@
+class TnApiException(Exception):
+    """Base class for all Trip Ninja SDK exceptions."""
+
+    def __init__(self, message: str, code: str = "TN_SDK_ERROR"):
```

Comment:
thoughts on starting a constants file for `TN_SDK_ERROR`?
Could put URLS / compression in there aswell maybe